src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example2.java

```diff
@@ -0,0 +1,45 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodLength">
+      <property name="tokens" value="METHOD_DEF"/>
+      <property name="max" value="4"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
+// xdoc section -- start
+public class Example2 {
+
+  // ok, CTOR_DEF is not in configured tokens
+  public Example2()  {
+    int var1 = 2;
+    int var2 = 4;
+    int sum = var1 + var2;
+  }
+
+  // violation below, 'Method firstMethod length is 6 lines (max allowed is 4)'
```

Comment:
Please add the same code to all examples
Here we missed
```
public Example1(int a)  {
    int var1 = 2;
    int sum = var1 + a;
  }
```

Reply:
@romani thanks for the reminder on this.

Reply:
@romani done

Reply:
@romani I tried to make all examples have the same code but for records in the first example I didn't put them in example 2,3 because in this case, both examples will exceed the file length limit and the compact ctor isn't the target of violation in this examples So, please tell me if this is acceptable or they should be exactly the same

Reply:
same for method `third method` it is useful in third example to check the spaces but if i put it in first the file length will exceed limit

Reply:
should I remove the record in the first Example and put the `thirdMethod` at this case the examples will be exactly the same but we will not test compact constuctors

Reply:
I might miss something but I see all examples to be almost the same and no violations of size exceed.